packages/ckeditor5-image/src/image/replaceimagesourcecommand.ts

```diff
@@ -34,6 +43,15 @@ export default class ReplaceImageSourceCommand extends Command {
 		this.value = this.isEnabled ? element.getAttribute( 'src' ) as string : null;
 	}
 
+	/**
+	 * Register callback which will be executed after command execution.
```

Comment:
It is not executed after command execution but during the execution. I would document the purpose of this (it cleans attributes not related to new source URL) and give some code example (i.e. how to remove `myId` attribute).